composer.json

```diff
@@ -11,7 +11,7 @@
     "minimum-stability": "dev",
     "require": {
         "php": ">=5.3.2",
-        "symfony/event-dispatcher": "^2.0 || ^3.0"
+        "symfony/event-dispatcher": "^2.3 || ^3.0"
```

Comment:
I can relate to this reason much easily. Thank for explaining @SpacePossum 

@jmikola Would you be open to bump versions with active security support for reasons I've talked about?